.github/workflows/ci.yml

```diff
@@ -100,10 +100,10 @@ jobs:
       run: |
         set -e
         set -x
-        apt-get --yes --quiet install --no-install-recommends gpg software-properties-common
+        apt-get --yes --quiet install --no-install-recommends gpg
         gpg --homedir="${PWD}/gnupg" --output /etc/apt/trusted.gpg.d/linuxcnc-deb-archive.gpg --export 3CB9FD148F374FEF
         DIST=$(echo ${{matrix.image}} | cut -d : -f 2)
-        add-apt-repository "deb http://linuxcnc.org $DIST base"
+        echo "deb http://linuxcnc.org $DIST base" >> /etc/apt/sources.list
```

Comment:
What about skipping the entire 'add extra repo' part on bookworm and unstable, is something like this:
```
case "$DIST" in
  bookwork|sid)
    ;;
  *)
     add_linuxcnc_org_repo
    ;;
esac
```